.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
+    branches: [ rework ]
+  pull_request:
+    branches: [ rework ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build & Test
+      run: |
+        yarn install
```

Comment:
Možná budeme teď tratit trochu času, ale bylo by fajn rozdělit příkazy do logických kroků a [sdílet data mezi kroky](https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts), tedy prvně install a pak třeba paralelně lint, test, lighthouse a další budoucí kontroly... Co si o tom myslíš?

Reply:
Vůbec bych se nebál tyhle věci dělat líně, tedy až v okamžiku, kdy pro to bude důvod. (Například pokud tam bude krok, který bude trvat a bude lepší ho pustit paralelně.)

Reply:
Tady mi jde i o určité pojmenování. Pokud je pipeline neúspěšná, jednoduše vím, která část musí být opravená. V zápětí bych rád ještě přidal Lighthouse check, tak nevím jestli to raději nerozdělit už teď...

Reply:
Instalaci závislostí a kontrolu kódu mně stejně dává smysl mít v jednom kroku, nedrobme ty procesy příliš. Lighthouse si bokem představit dovedu, navrhuju tohle mergnout a pro Lighthouse pak rozdělit, až ho budem přidávat.

Reply:
OK, to zní dobře.